hack/jenkins/common.sh

```diff
@@ -31,7 +31,8 @@ export KUBECONFIG="${TEST_HOME}/kubeconfig"
 export PATH=$PATH:"/usr/local/bin/:/usr/local/go/bin/:$GOPATH/bin"
 
 # installing golang so we could do go get for gopogh
-sudo ./installers/check_install_golang.sh "1.13.4" "/usr/local" || true
+WANT_GOLANG_VERSION=$(grep '^GO_VERSION' Makefile | awk '{ print $3 }')
+sudo ./installers/check_install_golang.sh "$WANT_GOLANG_VERSION" "/usr/local" || true
```

Comment:
there is another shell script with 
also change the version in this file:
hack/jenkins/minikube_cross_build_and_upload.sh

Reply:
oh this script is retrieving golang version from Makefile directly.

```bash
WANT_GOLANG_VERSION=$(grep '^GO_VERSION' Makefile | awk '{ print $3 }')
./hack/jenkins/installers/check_install_golang.sh $WANT_GOLANG_VERSION /usr/local
```